src/main/java/org/opentripplanner/common/model/RouteOriginDestination.java

```diff
@@ -0,0 +1,6 @@
+package org.opentripplanner.common.model;
+
+import java.io.Serializable;
+
+public record RouteOriginDestination(String route, String origin, String destination)
```

Comment:
Move to `org.opentripplanner.ext.fares.model`

Reply:
I think that would cause the FareRuleSet in  `org.opentripplanner.routing.core` to refer to this class in ext. Is that ok?

Reply:
I think it is ok, it already has a reference to `org.opentripplanner.ext.fares.model.FareAttribute`

Reply:
Actually, the entire `FareRuleSet` could be moved to `org.opentripplanner.ext.fares.model`